README.md

```diff
@@ -144,6 +144,8 @@ The cmdlet does not fully support the Inquire action for debug messages. Cmdlet
 ### Unreleased
 
 * Templates folder removed. Use the test templates in the [Tests.Template folder in the DSCResources repository](https://github.com/PowerShell/DscResources/tree/master/Tests.Template) instead.
```

Comment:
I don't see an example added, but I don't think I'm going to block on that. Please follow up with an example.